Basic-Car-Maintenance/Shared/Models/OdometerReading.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  OdometerReading.swift
+//  Basic-Car-Maintenance
+//
+//  Created by Nate Schaffner on 10/15/23.
+//
+
+import Foundation
+import FirebaseFirestoreSwift
```

Comment:
```suggestion
import FirebaseFirestoreSwift
import Foundation
```

Putting these in alphabetical order